test/converters/xcresult/xcresult.go

```diff
@@ -0,0 +1,188 @@
+package xcresult
```

Comment:
This converter is responsible for detecting *.xcresult files and parse it's TestSummaries.plist file, then convert it into a Junit4 xml format.

Reply:
Here you can see what structure needs to be parsed: https://github.com/bitrise-io/steps-deploy-to-bitrise-io/pull/79/files#diff-f1794a0dd4ef21c422ef14bd6f251acfR253